python/CONDUCT.rst

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
Does this duplicate what's in the root directory? If so, can we copy in a file rather than having multiple CONDUCT files?